cpp/src/strings/search/findall.cu

```diff
@@ -19,142 +19,124 @@
 
```

Comment:
Nothing actually has been modified in this file. The original `findall.cu` was deleted and this file was renamed to `findall.cu`. Github did not detect this and so is just highlighting the differences between the two files.